packages/react-magma-dom/src/components/Dropdown/DropdownContent.tsx

```diff
@@ -127,6 +131,19 @@ export const DropdownContent = React.forwardRef<
     }
   });
 
+  // For Expandable Dropdowns that don't require a max-height
+  let hasExpandable = false;
```

Comment:
I think this variable name and comment are throwing me off a bit. Could we be more verbose and call it `hasExpandableItems` or something?? and then the comment can also be updated